content/blog/2016/2016-05-12-sf-jam-jenkins-and-azure.adoc

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "SF JAM Report: Scaling Jenkins for Continuous Delivery with Azure"
+tags:
+- jam
+- azure
+- meetup
+author: rtyler
+---
+
+A few weeks ago, my colleague link:https://github.com/brianvdawson[Brian Dawson]
+and I were invited to present on
+link:http://www.meetup.com/jenkinsmeetup/events/228524373/[Scaling Jenkins for
+Continuous Delivery with Microsoft Azure] in Microsoft's
+link:http://microsoftreactor.com/about/[Reactor space]. Azure is Microsoft's
+public cloud offering and one of the many tools available to Jenkins users for
+adding elastic compute capacity, among other things, to their build/test/deploy
+infrastructure. While Brian's and my presentations are applicable to practically
+any cloud-based Jenkins environment, Thiago Almeida and Oguz Pastirmaci from
+Microsoft were also on-hand and presented some interesting Azure-specific
+offerings like
+link:https://azure.microsoft.com/en-us/services/container-service/[Azure
+Container Service] with Jenkins.
+
+To kick-off the sessions we asked attendees a few polling questions and
+received very telling responses:
+
+* How big is your Development/IT organization?
+* What is your role?
+* By show of hands do you practice?
+* At what scale (tooling and practice)?
+
+[quote, Brian Dawson]
+____
+The responses indicated that the majority of attendees were from small to
+medium organizations where they practiced Continuous Delivery across multiple
+teams. A notable 25% or greater attendees considered themselves "fullstack" or
+participating in all of the roles of Developer, QA, and Operations. Interesting
+when paired with the high number  (~80%) of the practicing CD.  Is it that
+modern team with fully implemented CD practices tend to blur the traditional
+lines of Developer, QA and Operations? In my experience this is the case for
+small to medium teams and companies but, not as much so when you begin to scale
+CD across large organizations.
```

Comment:
This is likely because modern teams, with mature CD practices, tend to blur the traditional
lines of Developer, QA and Operations. However, In my experience, while this is often the case for small to medium companies in large organizations team members tend to fall into the traditional roles, with CD providing the practice and platform to unify teams across roles.